test/Dataflow/demanded-bits10-invalid.opt

```diff
@@ -1,6 +1,6 @@
 
 
-; RUN: %souper-check %s > %t 2>&1
+; RUN: %souper-check -souper-use-alive %s > %t 2>&1
```

Comment:
I think in this instance (and a bunch of others) you've made alive testing better, but you have made testing of the klee backend worse, because you're changing the query

instead of removing the existing souper-check invocation, I think you should make this test case (and the others) contain two RUN: lines, one of which targets one backend, the other of which targets the other one